routing/src/main/kotlin/ru/surfstudio/compose/routing/NavigationDispatcher.kt

```diff
@@ -179,10 +189,20 @@ class NavigationDispatcher(
         navigatorCallback.remove()
     }
 
+    /**
+     * Check on back press custom callback
+     */
+    fun hasBackPressedCallback(): Boolean {
+        return onBackPressedCallback != null
+    }
+
     /**
      * Check is navigation stack empty
      */
     fun hasEnabledCallbacks(): Boolean {
+        if (onBackPressedCallback != null) {
+            return false
```

Comment:
```suggestion
            return true
```
? тк чистим onBackPressedCallback в clearData

Reply:
Разобралась, нужен false судя по описанию, тк стек не пуст если есть onBackPressedCallback (остался с помощью isHoldBackPressedCallback). Но тогда судя по всему название функции не соответствует содержанию, потому вопрос возник (hasEnabledCallbacks, при этом при наличии onBackPressedCallback вернет false)

Reply:
Обновил